docs/docs/clients/client-side/android.md

```diff
@@ -14,20 +14,23 @@ This SDK can be used for Android applications written in Kotlin. The source code
 
 ### Gradle
 
-Make sure your `settings.gradle` includes JitPack as a repository:
-
 ```groovy
 repositories {
     google()
     mavenCentral()
-    maven("https://jitpack.io")
 }
 ```
 
 In your project path `app/build.gradle` add a new dependency:
 
 <CodeBlock>{`implementation("com.flagsmith:flagsmith-kotlin-android-client:`}<AndroidVersion />"{`)`}</CodeBlock>
 
+:::info
+
+Note: jitpack support has been deprecated as of `v1.7.1`.
```

Comment:
I think we can leave this warning out. It's not relevant to new users, and new users don't need to do anything in response to it. Maybe they could remove JitPack as a repository if they're not using it anywhere else in their project, but it's not important either way.

Reply:
Done